### DISCUSSION TITLE
How to import YouTube videos through links into LosslessCut?

### DISCUSSION THREAD

**Discussion by Voxxel** [mifi/lossless-cut#1687] [Q&A] [answered]
Hi there, 
Is there a way to open YT videos by entering the video link? I can't find the field to do it. Tried from the Menu -> Open dialog by entering the link there but it gives an error message: "Unsupported File"

The youtube link I was tried to open is : 
https://youtu.be/syc78JzHGTs

**Comment by mifi** [comment, marked as answer]
Hi. this is not possible in losslesscut. you have to download the video yourself first. It might be possible to integrate https://github.com/yt-dlp/yt-dlp into losslesscut - however the legality of that is a bit hairy

**Comment by Voxxel** [comment]
Oh I see, thank you. Is this on to-do list or it's something really hard to achieve? Like telling the program to automatically choose the best quality available of the video from the YT link - for a headstart?

**Reply by mifi** [reply]
as i mentioned, that would involve integrating https://github.com/yt-dlp/yt-dlp into losslesscut, which could be a bit tricky because it's python-based. And due to possible legal issues, LosslessCut might never include that.
